internal/pkg/daemon/enricher/enricher.go

```diff
@@ -309,7 +309,7 @@ func (e *Enricher) addToBacklog(line *auditLine) error {
 		return errors.Wrap(setErr, "adding a line to the backlog")
 	}
 
-	e.logger.Info("line appended to backlog")
+	e.logger.V(1).Info("Line appended to backlog")
```

Comment:
Feel free to just remove this line instead. I thought I even removed it myself, I think it's just a leftover from the initial testing.

Reply:
Sure, removed that line